.eslintrc.json

```diff
@@ -66,7 +66,10 @@
         "@typescript-eslint/array-type": "off",
         "@typescript-eslint/ban-ts-comment": "off",
         "@typescript-eslint/ban-tslint-comment": "off",
-        "@typescript-eslint/ban-types": "off",
+        "@typescript-eslint/no-empty-object-type": "off",
```

Comment:
Was split from `ban-types` and `no-empty-interfaces` (https://github.com/typescript-eslint/typescript-eslint/releases/tag/v8.0.0)